pkg/cmd/pr/variables/variables.go

```diff
@@ -179,7 +179,9 @@ func (o *Options) modifyVariables(text string) error {
 func (o *Options) loadPRComments(envVars map[string]string, pr *scm.PullRequest) error {
 	ctx := o.GetContext()
 	prNumber := pr.Number
-	opts := scm.ListOptions{}
+	opts := scm.ListOptions{
```

Comment:
Is there any chance this will break for other scm providers?

Reply:
No, now only gitlab provider handles this parameter. ASC is the default in GitHub